docs/Tutorials/Syntax/Quest-Packages.md

```diff
@@ -0,0 +1,360 @@
+              questAlreadyDone:
+                text: "Hey! I don't need you anymore. Thanks again for the help."
+                conditions: "questDoneTag"
+            player_options:
+              probably:
+                text: "Yes I can do that for you! How much wood do you need?"
+                pointer: "woodAmountAnswer"
+              letsDoIt:
+                text: "Alright let's get the job done!"
+                pointer: "seeYou"
+                events: "questStarted"
+        ```
+    === "myEventsList1.yml"
+        ```YAML
+        {==events==}:
+          questStarted: "folder startedTagAdd,addWoodcuttingObj"
+          startedTagAdd: "tag add startedTag"
+          addWoodcuttingObj: "objective add woodCuttingObj"
+        ```
+    === "myEventsList2.yml"
+        ```YAML
+        {==events==}:
+          questDone: "folder takeWoodFromPlayer,rewardPlayer,addQuestDoneTag"
+          takeWoodFromPlayer: "take oakLog:10"
+          rewardPlayer: "give jonesAxe"
+          addQuestDoneTag: "tag add questDoneTag"
+          
+          addWoodcuttingDoneTag: "tag add woodcuttingDoneTag"
+        ```
+    === "myAwesomeObjectives.yml"
+        ```YAML
+        {==objectives==}:
+          woodCuttingObj: "block OAK_LOG -10 notify events:addWoodcuttingDoneTag"
+        ```
+    === "importantConditions.yml"
+        ```YAML
+        {==conditions==}:
+          woodcuttingDoneTag: "tag woodcuttingDoneTag"
+          logsInInventory: "item oakLog:10"
+          questDoneTag: "tag questDoneTag"
+          startedTag: "tag startedTag"
+        ```
+        
+As you can see: Every feature goes into a section like `events:`, `objectives:`, `conversations:` which are marked in
+{==blue==} in this example quest.
+Instead of file names like in 1.12, BetonQuest now uses these section names to understand the contents of a file.
+You can write these sections in any file you want, and it will still work! That's the way you can organize your quests.
+
+!!! note "A note about Sections"
+    While you can have multiple sections in one file, you can't have multiple sections with the same name.
+    For example, you can't have two `events:` sections in one file. If you do, the second one will overwrite the first one.
+    You can however have `events:` sections in two different files. In this case, the events from both files will be loaded.
+
+Download this example quest and play around with it. This will help you understand it.
+
+--8<-- "Tutorials/download-complete-files.md"
+    ```
+    /bq download BetonQuest/Quest-Tutorials main QuestPackages /Syntax/packageStructure/1-MultiFileStructure /packageStructure/MultiFile
+    ```
+    You can now find all files needed for this tutorial in this location:
+    "_YOUR-SERVER-LOCATION/plugins/BetonQuest/QuestPackages/packageStructure/MultiFile_"
+
+## 3. Creating a Quest Package with a single file
+
+Now that you understood how the new _multifile_ system works we will try another example.
```

Comment:
It is technically correct, that this feature is new, but i would not write it in tutorial, because it explains the feature, not if it is new or not. And this would also reduce mentanance.